### DISCUSSION TITLE
Anyone has a good idea to improve fps? My Android device has a gpu of low performance

### DISCUSSION THREAD

**Discussion by magicwo** [google/filament#4965] [Q&A]
Hi, I render a gltf model in my Android device and it has an animation。However , the FPS is only about 15fps. How can I improve the fps?

Android device:
Android 8.0
CPU Info : mt8167
CPU Arch: armeabi-v7a
CPU Core Num:4
GPU Info: PowerVR Rogue GE8300
Ram: 0.94 GB
Resolution:720x1440

**Comment by magicwo** [comment]
https://github.com/google/filament/files/7704288/fps15.html.zip
this is the systrace file

**Comment by magicwo** [comment]
PBR need lots of computation?

**Comment by bejado** [comment]
For starters, you should turn off [post-processing](https://github.com/google/filament/blob/main/filament/include/filament/View.h#L581) and see how much that improves performance. Set [render quality](https://github.com/google/filament/blob/main/filament/include/filament/View.h#L474) to `LOW`. You can also experiment with [dynamic resolution](https://github.com/google/filament/blob/main/filament/include/filament/View.h#L460), which renders the scene at a lower resolution and up-scales.
